.github/workflows/chromatic-deployment.yaml

```diff
@@ -27,6 +27,20 @@ jobs:
       - name: Install dependencies
         run: pnpm install
 
+      - name: Create .env file from secrets
+        shell: bash
+        working-directory: ./apps/docs
+        run: |
```

Comment:
Can some of this be abstracted and used across github actions? If not now, in the future?

Reply:
Yeah, I think so.
I'll add this suggestion to the CI refactor task that we already have in the backlog 👍